runTests.py

```diff
@@ -97,10 +97,12 @@ def makeTest(name, j):
     """Run the make command for a given single test."""
     doCommand('make -j%d %s' % (j or 1, name))
 
-def runTest(name, run_all=False):
+def runTest(name, run_all=False, mpi=False):
     executable = mungeName(name).replace("/", os.sep)
     xml = mungeName(name).replace(winsfx, "")
     command = '%s --gtest_output="xml:%s.xml"' % (executable, xml)
+    if mpi and "mpi_" in name:
+        command = "mpirun -np 2 " + command
```

Comment:
should we include a parameter to change that `2` to something that's specified by the user?

Reply:
making that configurable would be nice, but I would also be fine to do with this logic:

- at least 2
- if -j#CPUs is given and #CPUs is >2, then just take that number

We want at least two processes, but possibly more.

Reply:
Okay, will do @wds15 suggestion here